templates/flavors/apiserver-ilb/patches/kubeadm-config-template.yaml

```diff
@@ -8,6 +8,5 @@ spec:
       # /etc/hosts file is updated with a pre-created DNS name of the API server and internal load-balancer's IP.
       # This custom DNS Resolution of the API server ensures that the worker nodes can reach the API server when
       # the public IP of the API server is not accessible.
-      # 10.0.0.100 is the default IP that gets assigned to an internal load balancer.
       preKubeadmCommands:
-        - echo '10.0.0.100   ${CLUSTER_NAME}-${APISERVER_LB_DNS_SUFFIX}.${AZURE_LOCATION}.cloudapp.azure.com' >> /etc/hosts
+        - echo '30.0.0.100   ${CLUSTER_NAME}-${APISERVER_LB_DNS_SUFFIX}.${AZURE_LOCATION}.cloudapp.azure.com' >> /etc/hosts
```

Comment:
For my own understanding, why is this changed to 30?

Reply:
`10.0.0.100` is the default value for internal ILB IP Address. Ref: https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/e0f2b44aec58ed04bbd754a98bace470bea32fff/api/v1beta1/azurecluster_default.go#L45

A non default value, `30.0.0.100`, will help in spotting runtime or user errors. 
Also, using `30.0.0.100` supports the goal of this PR; lets user configure the IP of the ILB.

Reply:
if we wanna demonstrate a non-default private IP address value, we want to choose a private IP address

Ref:

- https://www.iana.org/help/private-addresses

(unless I'm misunderstanding something)